test/integration/DeclarationStatusConnectorSpec.scala

```diff
@@ -84,27 +84,6 @@ class DeclarationStatusConnectorSpec extends IntegrationTestSpec
       await(sendValidXml())
       verifyBackendDecServiceWasCalledWith(requestBody = expectedStatusPayloadRequest.toString(), maybeUnexpectedAuthToken = Some(incomingAuthToken))
     }
-
```

Comment:
Are these tests no longer needed?

Reply:
That's right, because exceptions are no longer used to represent errors.